query-translator/src/types.rs

```diff
@@ -25,37 +27,57 @@ use mentat_sql::{
     QueryBuilder,
     QueryFragment,
     SQLiteQueryBuilder,
+    SQLQuery,
 };
 
 //---------------------------------------------------------
 // A Mentat-focused representation of a SQL query.
 
-enum ColumnOrExpression {
+pub enum ColumnOrExpression {
     Column(QualifiedAlias),
-    Integer(i64),             // Because it's so common.
+    Integer(i64),            // Because it's so common.
+    Value(TypedValue),       // TODO: replace with a SQLite value.
```

Comment:
I don't have a strong preference.  On the transactor side of the house, I am brewing a db (concrete storage) vs. tx (Mentat transaction processing) split, and I see some value in separating the two concerns.  It's not clear to me where the "SQL data from here out" is, exactly, in the query flow, so I'm happy to leave this as TODO.